src/test/kotlin/pl/touk/krush/BookTest.kt

```diff
@@ -44,4 +47,11 @@ class BookTest {
         }
     }
 
+    /**
+     * comment
+     */
+    private fun a_test() {
```

Comment:
\[detekt\] INFO: \[empty\-blocks/EmptyFunctionBlock\] Empty block of code detected\. As they serve no purpose they should be removed\.